generate-deb.sh

```diff
@@ -432,16 +432,16 @@ Source: dropbox
 Section: gnome
 Priority: optional
 Maintainer: Dropbox <support@dropbox.com>
-Build-Depends: cdbs, debhelper (>= 9), build-essential, libnautilus-extension-dev (>= 3.10.1), libglib2.0-dev (>= 2.40), python-gi (>= 3.12), python-docutils
+Build-Depends: cdbs, debhelper (>= 9), build-essential, libnautilus-extension-dev (>= 3.10.1), libglib2.0-dev (>= 2.40), python-gi (>= 3.12), python-docutils, python3-gi (>= 3.12), python3-docutils
 Standards-Version: 3.9.4.0
 
 Package: dropbox
 Replaces: nautilus-dropbox
 Breaks: nautilus-dropbox
 Provides: nautilus-dropbox
 Architecture: any
-Depends: procps, python-gi (>= 3.12), python (>= 2.7), \${python:Depends}, \${misc:Depends}, libatk1.0-0 (>= 2.10), libc6 (>= 2.19), libcairo2 (>= 1.13), libglib2.0-0 (>= 2.40), libgtk-3-0 (>= 3.10.8), libpango1.0-0 (>= 1.36.3), lsb-release, gir1.2-gdkpixbuf-2.0 (>= 2.30.7), gir1.2-glib-2.0 (>= 1.40.0), gir1.2-gtk-3.0 (>= 3.10.8), gir1.2-pango-1.0 (>= 1.36.3)
-Suggests: nautilus (>= 3.10.1), python-gpg (>= 1.8.0)
+Depends: procps, python3-gi (>= 3.12), python3 (>= 3.4.0), \${python:Depends}, \${misc:Depends}, libatk1.0-0 (>= 2.10), libc6 (>= 2.19), libcairo2 (>= 1.13), libglib2.0-0 (>= 2.40), libgtk-3-0 (>= 3.10.8), libpango1.0-0 (>= 1.36.3), lsb-release, gir1.2-gdkpixbuf-2.0 (>= 2.30.7), gir1.2-glib-2.0 (>= 1.40.0), gir1.2-gtk-3.0 (>= 3.10.8), gir1.2-pango-1.0 (>= 1.36.3)
+Suggests: nautilus (>= 3.10.1), python3-gpgme (>= 0.3)
```

Comment:
We want to suggest python3-gpg, which is newer than python3-gpgme (we got rid of python3-gpgme last year b/c it's obsolete). It's not showing up in ubuntu's package search right now b/c the index is broken for releases other than trusty. So this should be `python3-gpg (>= 1.8.0)`

Reply:
Thanks! I was wondering why `python3-gpg` was not exposed on Ubuntu package list.